server/conf/evolutions/default/47.sql

```diff
@@ -0,0 +1,8 @@
+# --- !Ups
+
+alter table programs add column program_type varchar;
+update programs set program_type = 'DEFAULT' where program_type is null;
```

Comment:
**question (blocking):** why two statements rather than `alter table programs add column program_type varchar default 'DEFAULT';`?

Reply:
switched to one statement.